Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.h

```diff
@@ -350,13 +348,11 @@ ALLOW_NEW_API_WITHOUT_GUARDS_END
     RetainPtr<AVSampleBufferDisplayLayer> m_sampleBufferDisplayLayer;
     RetainPtr<AVSampleBufferVideoRenderer> m_sampleBufferVideoRenderer;
 
-    struct AudioRendererProperties {
-        bool hasAudibleSample { false };
-    };
 ALLOW_NEW_API_WITHOUT_GUARDS_BEGIN
-    HashMap<RetainPtr<CFTypeRef>, AudioRendererProperties> m_sampleBufferAudioRendererMap;
+    Vector<RetainPtr<AVSampleBufferAudioRenderer>> m_sampleBufferAudioRenderers;
```

Comment:
`HashSet<RetainPtr<AVSampleBufferAudioRenderer>>` is probably what would make most sense.